images/build/Dockerfiles/Dockerfile

```diff
@@ -21,7 +21,6 @@ RUN ./build.sh dotnetcore /opt/startupcmdgen/dotnet
 RUN ./build.sh node       /opt/startupcmdgen/nodejs
 RUN ./build.sh php        /opt/startupcmdgen/php
 RUN ./build.sh python     /opt/startupcmdgen/python
-RUN ./build.sh golang     /opt/startupcmdgen/golang
```

Comment:
this one was causing problem ... not sure why this was added ... this is for the runtime scriptgenerator ..we are only focused about buildscript generator for golang

Reply:
Ah got it! Looks like it was from my very initial commits while I was referencing Ruby. Thank you for catching this! 👍